grades/apps.py

```diff
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
+
+
+class CourseConfig(AppConfig):
+    name = 'course'
```

Comment:
I think that there should be a correlation between the names here and in settings.py and that you don't need an app per model class

Reply:
@Yarboa @ifireball @simzacks can you clarify what is expected here?

Reply:
@ravidbro I forgot to add course to the settings.py I add it now. I did the dame as Gal and Noa did in their files. They both add an admin.py and apps.py as I worte it.

Reply:
@ravidbro  If you think that we all need to merge our apps.py file and it is the right way to do it i will merge it with Gal's file and move the file into the correct position where it should be, cause right now we all have our files under our feature folder. I  think it connects to what we talked about in our conversation that every feature represent as another app.

Reply:
about apps.py - since you defined each feature as a different app (which is a valid option), each one of you should have it's own apps.py
but you should have only one AppConfig and for some reason, you have here two.
about admin.py - I'm not sure how 'apps.get_models()', if it brings models from all the apps or only from yours, you can test it or put an explicit list instead.

Reply:
@ravidbro  I think I get it now correct me if I'm wrong. In the AppConfig I only put the app feature I did? for example The main feature is to have a course list with grades. So in my case the AppConfig should include "CourseGradeBordConfig" only  and not the all classes as I made it?

Reply:
AppConfig per app/feature, not per model class

Reply:
ok thats what Im changing it now. And olso I read some documetetion I think I add a mistake on the settings.py file.
I wrote the classes name that I made but in the documetetion it's write like this "CourseGradeBordConfig.apps.CourseGradeBordConfig". what is the right way?

Reply:
although it probably works when writing only the class name of the app, it's better to write the full package module, but it's not 'CourseGradeBordConfig.apps.CourseGradeBordConfig' because your package name is 'grades' so it should be 'grades.apps.CourseGradeBordConfig'

Reply:
yes you are right. Im updating my PR according to this now.

Reply:
I saw that you deleted the content of admin.py.
did you test that it's not needed?